### ISSUE TITLE
Comment engine?

### ISSUE THREAD

**Issue by vak** [aaaakshat/gatsby-starter-texblog#1]
interesting project, thank you!

is it possible to add some comment engines?

**Comment by bitmaks** [comment]
That's a good idea. I'm not sure how comment engines would work on a static site, I may look into this in the future — seems like a good enhancement.

**Comment by vak** [comment]
it works. I was about to add Disqus, but then I saw this: https://trends.google.com/trends/explore?date=all&q=disqus

smth happend to it near 2014 and it started to die. Its killer doesn't seem to be Spot.im, htmlcommentbox, commento, jetpack or even facebook comments... not really clear what to choose

**Comment by bitmaks** [comment]
Yeah, I was looking online and I found this link from 2014 — around the time when the fall seems to have begun — [https://techcrunch.com/2014/04/07/disqus-sponsored-comments/]() in recent years Disqus has started bombarding the free accounts with ads so I think more people have decided to just remove comments entirely rather than stay with Disqus.

I'm still looking into it, but right now it seems that static sites aren't that great for third party comment systems because of the hassle of hosting your own backend. [This implementation guide](https://www.gatsbyjs.com/docs/how-to/adding-common-features/adding-comments/) by the Gatsby team shows some of the larger ones like Facebook comments, Commento and Disqus, but I'm still hesitant to include them.

If there is interest I could possibly create push a version with a demo implementation.
